lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -97,16 +80,22 @@
 '''
 
 EXAMPLES = '''
-# Basic server certificate upload
-tasks:
-- name: Upload Certificate
-  iam_cert:
+# Basic server certificate upload from local file
+- iam_cert:
     name: very_ssl
     state: present
-    cert: somecert.pem
-    key: privcertkey
-    cert_chain: myverytrustedchain
+    cert: "{{ lookup('file', 'path/to/cert') }}"
```

Comment:
See https://github.com/ansible/ansible/pull/20787 for another alternative, which was since removed.